runtime/compiler/control/J9Options.cpp

```diff
@@ -2310,6 +2310,13 @@ J9::Options::setupJITaaSOptions()
          // no IProfiler info is collected at the server itself
          self()->setOption(TR_DisableIProfilerThread);
          }
+
+      // In the JITaaS world, expensive compilations are performed remotely so there is no risk of blowing the footprint limit on the JVM
+      // Setting _expensiveCompWeight to a large value so that JSR292/hot/scorching compilation are allowed to be executed concurrently
+      if (TR::Options::_expensiveCompWeight < 101)
+         {
+         TR::Options::_expensiveCompWeight = 101; // Scorching weight = 100
+         }
```

Comment:
Can we just set it to `MAX_WEIGHT` without checking anything?

Reply:
set to `MAX_WEIGHT`, used 255 instead of `MAX_WEIGHT` to avoid including the `CompilationRuntime.hpp` header in `J9Options.cpp`.

Reply:
Use `MAX_WEIGHT` and don't worry about the header, it's already included in that file.

Reply:
Changed as requested